tasmota/include/tasmota_template.h

```diff
@@ -198,6 +198,9 @@ enum UserSelectablePins {
   GPIO_ADE7953_RST,                    // ADE7953 Reset
   GPIO_NRG_MBS_TX, GPIO_NRG_MBS_RX,    // Generic Energy Modbus device
   GPIO_ADE7953_CS,                     // ADE7953 SPI Chip Select
+#ifdef ESP32
```

Comment:
The numbers should not depend on compilation option. Please remove #ifdef

Reply:
But I see several similar ones above

Reply:
Oh yes. Sorry my bad. I misread as a Dali I def. Please ignore my comment